test/mysql.test.js

```diff
@@ -633,6 +633,15 @@ describe('mysql', function() {
               });
             });
       });
+
+      it('filter with case sensitive regex string', function(done) {
```

Comment:
I see what you mean. I also thought about doing that but those tests are already covered. Look:
https://github.com/strongloop/loopback-connector-mysql/blob/master/test/mysql.test.js#L586
https://github.com/strongloop/loopback-connector-mysql/blob/master/test/mysql.test.js#L703